queue_services/entity-emailer/src/entity_emailer/email_processors/__init__.py

```diff
@@ -206,5 +207,10 @@ def get_jurisdictions(identifier: str, token: str) -> str:
     response = requests.get(
         f'{current_app.config.get("LEGAL_API_URL")}/mras/{identifier}', headers=headers
     )
-
-    return response
+    if response.status_code != HTTPStatus.OK:
+        return None
+    try:
+        return response.json()
+    except Exception:  # noqa B902; pylint: disable=W0703;
+        current_app.logger.error('Failed to get MRAS response')
+        return None
```

Comment:
thanks for fixing this.  when you have a minute, can you add a comment to #22435 directly too?  Maybe when emailer gets upgrade to v2, whoever picks up #22345 can just do a quick test to verify MRAS flow is working before moving it to RFQA